src/UserGuide/Master/stage/Delete-Data/TTL.md

```diff
@@ -21,29 +21,42 @@
 
 # TTL
 
-IoTDB supports storage-level TTL settings, which means it is able to delete old data automatically and periodically. The benefit of using TTL is that hopefully you can control the total disk space usage and prevent the machine from running out of disks. Moreover, the query performance may downgrade as the total number of files goes up and the memory usage also increase as there are more files. Timely removing such files helps to keep at a high query performance level and reduce memory usage.
+IoTDB supports device-level TTL settings, which means it is able to delete old data automatically and periodically. The benefit of using TTL is that hopefully you can control the total disk space usage and prevent the machine from running out of disks. Moreover, the query performance may downgrade as the total number of files goes up and the memory usage also increase as there are more files. Timely removing such files helps to keep at a high query performance level and reduce memory usage.
 
 The default unit of TTL is milliseconds. If the time precision in the configuration file changes to another, the TTL is still set to milliseconds.
 
-## Set TTL
-
-The SQL Statement for setting TTL is as follow:
-
-```
-IoTDB> set ttl to root.ln 3600000
-```
+When setting TTL, the system will look for all devices included in the set path and set TTL time for these devices. The system will delete expired data at the device granularity.
+After the device data expires, it will not be queryable, but the data in the disk file cannot be guaranteed to be deleted immediately (it will be deleted within a certain time), but it can be guaranteed to be deleted eventually.
```

Comment:
"(it will be deleted within a certain time)" seems confusing, may should remove it.
"cannot be guaranteed to be deleted immediately, but it can be guaranteed to be deleted eventually."
And, there are two "but"s in this sentence, break it into two sentences.